source/process/training.rst

```diff
@@ -106,7 +106,66 @@ Mattermost Staff Meetings
 
 For confidentiality of customers, internal process meetings are limited to Mattermost staff only.
 
+Customer Obsession All-Hands Meeting
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Attendees:
+
+- All Mattermost Staff
+
+Objectives:
+
+- Share important company-wide updates with staff across the organization.
+- Educate staff on key customer use cases and feature requests.
+
+Time:
+
+- Weekly meeting on Wednesdays from 8:00am to 8:30am Palo Alto time.
+
+Procedure:
+
+**Before meeting:**
+
+1. (Chair) Schedules items for the weekly agenda, includes company update from CEO or CTO, and customer updates.
```

Comment:
The job of CEO is to lead and empower the company's leaders. CEO should do the introduction and unless there's something special, department heads should share the updates.

Reply:
You are also giving an update tomorrow (about YC). So how about company updates are from "leads"?

0/5, "leads" may not be the correct term here.